dmscripts/data_retention_remove_user_data.py

```diff
@@ -20,40 +20,40 @@ def data_retention_remove_user_data(
     for user in all_users:
         last_logged_in_at = datetime.strptime(user['loggedInAt'], DATETIME_FORMAT)
         if last_logged_in_at < cutoff_date:
-            if dm_mailchimp_client is not None:
-                email_hash = dm_mailchimp_client.get_email_hash(user["emailAddress"])
-                logger.info(
-                    "Checking mailing list membership for email with hash %s (user %s)",
-                    email_hash,
-                    user["id"],
-                )
-                mailing_lists = dm_mailchimp_client.get_lists_for_email(user["emailAddress"])
-                if mailing_lists:
-                    for mailing_list in mailing_lists:
-                        logger.warn(
-                            f"%sRemoving email with hash %s from list %s ('%s')",
-                            prefix,
-                            email_hash,
-                            mailing_list["list_id"],
-                            mailing_list["name"],
-                        )
-                        if not dry_run:
-                            rm_result = dm_mailchimp_client.permanently_remove_email_from_list(
-                                email_address=user["emailAddress"],
-                                list_id=mailing_list["list_id"],
-                            )
-                            if not rm_result:
-                                raise MailchimpRemovalFailed(
-                                    "Mailchimp failure trying to permanently_remove_email_from_list"
-                                )
-                else:
+            if not user['personalDataRemoved']:
```

Comment:
We can filter our users by this on the API I think - `data_api_client.find_users_iter(personal_data_removed=False)`?

Reply:
Done

Reply:
Er.. did you push the commit?

Reply:
The question is - did I push the _right_ commit?